core/constants.py

```diff
@@ -116,8 +127,10 @@ def __getattr__(self, name: str) -> Any:
         return self[name]
 
 
-constants = Constants(parse_json_from_ts(  # pylint:disable=invalid-name
-    get_package_file_contents('assets', 'constants.ts')))
+package_content = get_package_file_contents('assets', 'constants.ts')
+# Ruling out the possibility of different types for mypy type checking.
+assert isinstance(package_content, str)
+constants = Constants(parse_json_from_ts(package_content)) # pylint:disable=invalid-name
```

Comment:
The `assert` should no longer be needed when you implement the changes above.

Reply:
Done